samples/javascript_nodejs/21.luis-with-appinsights/index.js

```diff
@@ -4,20 +4,21 @@
 const path = require('path');
 const restify = require('restify');
 const { BotFrameworkAdapter } = require('botbuilder');
+const { ApplicationInsightsTelemetryClient, ApplicationInsightsWebserverMiddleware } = require('botbuilder-applicationinsights');
 const { BotConfiguration } = require('botframework-config');
 const { LuisBot } = require('./bot');
 const { MyAppInsightsMiddleware } = require('./middleware');
 
 // Read botFilePath and botFileSecret from .env file.
 // Note: Ensure you have a .env file and include botFilePath and botFileSecret.
 const ENV_FILE = path.join(__dirname, './.env');
-require('dotenv').config({ path: ENV_FILE });
+const env = require('dotenv').config({ path: ENV_FILE });
```

Comment:
I removed the 'const env =' code because it's not used in the sample. the code assumes the process.env.